### PR TITLE
LPS-104414 Repeatable web content structure items causes error and block publishing

### PR DESCRIPTION
**Pull request by NemethNorbert**
Hey @natocesarrego,

[LPS-104414](https://issues.liferay.com/browse/LPS-104414),

Please review it,

Thanks

### CODE REVIEW

**Comment by NemethNorbert** [comment]
ci:test:sf

**Comment by NemethNorbert** [comment]
ci:test:relevant

**Comment by NemethNorbert** [comment]
ci:test:sf

**Comment by NemethNorbert** [comment]
ci:test:sf

**Comment by NemethNorbert** [comment]
ci:test:relevant

**Comment by NemethNorbert** [comment]
ci:test:sf

**Comment by NemethNorbert** [comment]
ci:test:sf

**Comment by NemethNorbert** [comment]
ci:test relevant

**Comment by natocesarrego** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js [hunk 1]
```diff
@@ -4105,9 +4105,16 @@ AUI.add(
 					var newFieldLocalizations = repeatedField.get(
 						'localizationMap'
 					);
-					var totalLocalizations = originalField.get(
-						'localizationMap'
-					);
+
+					var totalLocalizations;
+
+					if (originalField) {
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js]: Hi @NemethNorbert,

In which use case the original field is `undefined`?

Comment (NemethNorbert) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js]: Original field is undefined, when newNestedFields.length is higher then originalNestedFields.length. In this scenario, instance.populateBlankLocalizationMap will be called with an undefined originalNestedFields

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by natocesarrego** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/81134. [See changes here.](https://github.com/natocesarrego/liferay-portal/compare/0fc83ef95c53~1...0fc83ef95c53) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**